recipes/src/main/resources/quarkus-updates/core/3.7.yaml

```diff
@@ -16,4 +16,23 @@ recipeList:
   - org.openrewrite.maven.UpgradePluginVersion:
       groupId: org.apache.maven.plugins
       artifactId: maven-compiler-plugin
-      newVersion: 3.12.1
+      newVersion: 3.12.0
+  - org.openrewrite.maven.ChangePropertyValue:
+      key: compiler-plugin.version
+      newValue: 3.12.0
+  - org.openrewrite.maven.UpgradePluginVersion:
+      groupId: org.apache.maven.plugins
+      artifactId: maven-surefire-plugin
+      newVersion: 3.2.3
+  - org.openrewrite.maven.ChangePropertyValue:
```

Comment:
Updating the property will be dropped when we update to an OpenRewrite version with the fix.